.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: ruby
+language: minimal
```

Comment:
Why you changed language name?

Reply:
since we're using docker now we don't need Travis to install ruby